src/components/Progressbar/BProgressBar.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import "react-step-progress-bar/styles.css";
+import { ProgressBar } from "react-step-progress-bar";
+import Grid from '@material-ui/core/Grid';
+
+function BProgressBar(){
+  return(
+    <div>
+      <Grid container 
+      spacing = {1}
+      justify ="center" 
+      alignItems = "center"
+      >
+      <Grid
+        item align = "center"
+        item xs ={5} sm={5} md={5} lg={5}>
+        Bias Score
+      </Grid>
+      <Grid 
+        style={{border: "2px solid red",borderRadius:"2000px"}}
```

Comment:
Styling looking good, but I'd like to suggest that the gradient goes from red to green rather than green to red. 
This keeps it consistent with the gauges and colour psychology.

Reply:
sure we can discuss this on call as well. we could match with  polarity and subjectivity gauges if possible as these two falls in the same category